config.default.php

```diff
@@ -237,3 +237,10 @@
 
 // TOTP (2FA) issuer name
 define('TOTP_ISSUER', 'Kanboard');
+
+// default filter expression
+define('DEFAULT_FILTER', 'status:open');
+
+// an expression that will be added to the predefined filters
+// define('ADDITIONAL_FILTER', 'board_date:<=today');  // use this when plugin TaskBoardDate is installed
+define('ADDITIONAL_FILTER', '');  // use this otherwise
```

Comment:
In my view, the best option would be to leave this to the plugin, but I am not PHP-savvy enough to solve that...

including a mechanism that detects a plugin into kanboard's config is not really clean, in my opinion. It would ruin the separation between kanboard itself and plugins. On the other hand it would be a lot more convenient...